registry/handlers/context.go

```diff
@@ -45,8 +45,8 @@ func getName(ctx context.Context) (name string) {
 	return ctxu.GetStringValue(ctx, "vars.name")
 }
 
-func getTag(ctx context.Context) (tag string) {
-	return ctxu.GetStringValue(ctx, "vars.tag")
+func getReference(ctx context.Context) (reference string) {
+	return ctxu.GetStringValue(ctx, "vars.reference")
```

Comment:
I still don't see this value getting set, if it is supposed to be set by mux it is not.

~~~
INFO[0386] response completed                            app.id=66e8ee98-cc08-46d8-a99f-7c1aeabf9e8d http.request.host=localhost:5001 http.request.id=1c182fa2-fa29-4fe1-8304-2a2cb41816a7 http.request.method=PUT http.request.remoteaddr=127.0.0.1:48976 http.request.uri=/v2/dmcgowan/hello/manifests/anything http.request.useragent=docker/1.5.0-dev go/go1.4.1 git-commit/ed64cc9 kernel/3.18.5-101.fc20.x86_64 os/linux arch/amd64 http.response.duration=8.457093ms http.response.written=0 vars.name=dmcgowan/hello version=v2.0.0-alpha.1+unknown
~~~